docker/docker-chaos/metrictank.ini

```diff
@@ -470,6 +470,8 @@ tag-support = true
 meta-tag-support = false
 # number of workers to spin up to evaluate tag queries
 tag-query-workers = 5
+# max allowed runtime for any single tag query
+tag-query-timeout = 5m
```

Comment:
that seems a bit high as a default, no? I can't imagine anybody opening grafana and waiting for 5min for the data to show

Reply:
It's quite high, but it's currently "unlimited". Grafana isn't the only use case, I didn't want to break anyone that would be doing some super expensive index ops successfully today

Reply:
ok, let's leave it at that then. if someone wants to it's easy to just lower the setting anyway.